packages/store/configure.tsx

```diff
@@ -1,22 +1,22 @@
-import { createStore, applyMiddleware, compose } from 'redux'
-import { ApolloClient, createNetworkInterface, gql, IntrospectionFragmentMatcher } from 'react-apollo'
-import { routerMiddleware } from 'react-router-redux'
-import thunk from 'redux-thunk'
-import createSagaMiddleware from 'redux-saga'
-import * as persistState from 'redux-localstorage'
+import { createStore, applyMiddleware, compose } from "redux"
```

Comment:
Include the semicolon as well on linter fixes (for entire file)

Reply:
resolved